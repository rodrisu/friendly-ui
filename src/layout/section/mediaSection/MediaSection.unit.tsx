import React from 'react'
import renderer from 'react-test-renderer'

import { MediaSection } from './index'

describe('MediaSection', () => {
  it('should render MediaSection', () => {
    const section = <MediaSection>default section</MediaSection>
    const renderedSection = renderer.create(section).toJSON()
    expect(renderedSection).toMatchSnapshot()
  })
})
