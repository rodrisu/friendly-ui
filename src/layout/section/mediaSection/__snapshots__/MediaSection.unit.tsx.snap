// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaSection should render MediaSection 1`] = `
@media (min-width:800px) {
  .c0 {
    padding-left: 24px;
    padding-right: 24px;
    margin-left: auto;
    margin-right: auto;
    max-width: 662px;
  }
}

<div
  className="c0 media-section"
  role="presentation"
>
  default section
</div>
`;
